src/context/Provider.js

```diff
@@ -4,12 +4,26 @@ import PropTypes from 'prop-types';
 import initialState from './initialState';
 import globalReducer from './reducer';
 
+const getCountry = async () => {
+  return fetch('http://ip-api.com/json')
+    .then((res) => res.json())
+    .then((res) => {
+      localStorage.setItem('country', JSON.stringify(res.country));
+      return res.country;
+    })
+    .catch(() => {
+      const defaultCountry = 'Canada';
+      localStorage.setItem('country', JSON.stringify(defaultCountry));
+      return defaultCountry;
+    });
+};
+
 let localState;
 
 if (typeof window !== `undefined`) {
   localState = JSON.parse(localStorage.getItem('cart-items'));
-  if (localState) {
-    localState.visitedPage = [];
+  if (localState && localState.version !== '1.0.0') {
```

Comment:
we should abstract the version into an environment variable, so then we update the .env file moving forward and don't have to keep track of this in the code, i feel its more standard that way
ie. localState.version !== ENV.version

Reply:
yes that is a good point, will add the change!